src/main/java/svc/controllers/NotFoundException.java

```diff
@@ -0,0 +1,32 @@
+package svc.controllers;
+
+public class NotFoundException extends Exception {
```

Comment:
You should be able to just extend RuntimeException and I think there is less boilerplate code to write...